templates/base.html.twig

```diff
@@ -1,34 +1,44 @@
 <!DOCTYPE html>
 <html lang="fr">
-	<head>
-		<meta charset="UTF-8">
-		<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-		<link rel="icon" type="image/png" href="{{ asset('build/images/logo-wild-code-school.svg') }}"/>
-		<title>
-			{% block title %}Mocktail Celeste |
-			{% endblock %}
-		</title>
-		{# Run `composer require symfony/webpack-encore-bundle` to start using Symfony UX #}
-		{% block stylesheets %}
-			<link rel="icon" type="image/png" href="../assets/images/logo.png"/>
-			<link rel="stylesheet" href="//use.fontawesome.com/releases/v5.0.7/css/all.css"/>
-			<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css" rel="stylesheet"/>
-			{{ encore_entry_link_tags('app') }}
-		{% endblock %}
-
-		{% block javascripts %}
-			<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js"></script>
-			<script src="assets/js/navbar.js"></script>
-			{{ encore_entry_script_tags('app') }}
-		{% endblock %}
-	</head>
-	<body>
-		{% block navbar %}
-			{% include "partials/_navbar.html.twig" %}
-		{% endblock %}
-		{% block content %}{% endblock %}
-		{% block footer %}
-			{% include "partials/_footer.html.twig" %}
-		{% endblock %}
-	</body>
-</html>
\ No newline at end of file
+    <head>
+        <meta charset="UTF-8" />
+        <meta
+            name="viewport"
+            content="width=device-width, initial-scale=1, shrink-to-fit=no"
+        />
+        <link
+            rel="icon"
+            type="image/png"
+            href="{{ asset('build/images/logo-wild-code-school.svg') }}"
+        />
+        <title>{% block title %}Mocktail Celeste | {% endblock %}</title>
+        {# Run `composer require symfony/webpack-encore-bundle` to start using
+        Symfony UX #} {% block stylesheets %}
+        <link rel="icon" type="image/png" href="build/images/logo.png" />
+        <link
+            rel="stylesheet"
+            href="//use.fontawesome.com/releases/v5.0.7/css/all.css"
+        />
+        <link
+            href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css"
+            rel="stylesheet"
+        />
+        {{
+            encore_entry_link_tags("app")
+        }}
+        {% endblock %} {% block javascripts %}
+        <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js"></script>
+        <script src="assets/js/navbar.js"></script>
+        {{
+            encore_entry_script_tags("app")
+        }}
+        {% endblock %}
+    </head>
+    <body>
+        {% block navbar %} {% include "partials/_navbar.html.twig" %} {%
+        endblock %} {% block content %}
+        <main></main>
```

Comment:
wait for your PR "update Mocktail controller and index" to be merged on dev then pull dev on this branch to remedy the main tag issue (or solve it here)